aggregator/src/test/scala/za/co/absa/spline/ol/aggregator/SystemInfoExtractorSpec.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2022 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spline.ol.aggregator
+
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+import za.co.absa.spline.ol.model.openlineage.v0_3_1.RunEvent
+
+class SystemInfoExtractorSpec extends AnyFlatSpec with Matchers {
+
+  val emptyRunEvent = RunEvent(None, null, null, null, None, None, "", "")
+
+  it should "recognize open lineage spark integrations" in {
+    val event = emptyRunEvent.copy(producer = "https://github.com/OpenLineage/OpenLineage/tree/0.6.1/integration/spark")
+    val result = SystemInfoExtractor.extract(event)
+    result.name shouldBe "spark"
```

Comment:
minor: although in this case both `shouldBe` and `shouldEqual` are interchangeable the latter is semantically closer to string comparison and is read better.  The word `shouldBe` suits better comparing primitive types or checking predicates, e.g.:
```scala
x shouldBe 5
x shouldBe > (5)
opt shouldBe defined
list shouldBe empty

// but

s shouldEqual "foobar"
seq shoudlEqual Seq(1, 2, 3)
```